test/unit/parse-siti-payment-file/build-payment-file.test.js

```diff
@@ -5,87 +5,115 @@ jest.mock('../../../app/parse-siti-payment-file/validate')
 const validate = require('../../../app/parse-siti-payment-file/validate')
 
 const { buildAndTransformParseFile } = require('../../../app/parse-siti-payment-file/build-payment-file')
+const { LUMP_SUMS, SFI_PILOT, SFI } = require('../../../app/schemes')
 
 let fileBuffer
-let sequence
+let schemeType
 
 let batchHeaders
-let batchPaymentRequests
+let batchPaymentRequestsSFI
+let batchPaymentRequestsLumpSums
 
 describe('SITI payment file batch header is split into batches, headers and lines', () => {
   beforeEach(async () => {
     fileBuffer = Buffer.from('B^2021-08-12^2^200^0001^SFIP^AP\r\nH^SFI00000001^01^SFIP000001^1^1000000001^GBP^100^RP00^GBP^SFIP^M12\r\nL^SFI00000001^100^2022^80001^DRD10^SIP00000000001^RP00^N^1^G00 - Gross value of claim^2022-12-01^2022-12-01^SOS27\r\nH^SFI00000002^03^SFIP000002^2^1000000002^GBP^100^RP00^GBP^SFIP^M12\r\nL^SFI00000002^100^2022^80001^DRD10^SIP00000000002^RP00^N^1^G00 - Gross value of claim^2022-12-01^2022-12-01^SOS273\r\n')
-    sequence = '0001'
-
-    batchHeaders = [
-      {
-        batchValue: 200,
-        exportDate: '2021-08-12',
-        ledger: 'AP',
-        numberOfPaymentRequests: 2,
-        sequence: '0001',
-        sourceSystem: 'SFIP'
-      }
-    ]
-    batchPaymentRequests = [
-      {
-        contractNumber: 'SFIP000001',
-        currency: 'GBP',
+    schemeType = {
+      batchId: '0001',
+      scheme: SFI_PILOT
+    }
+
+    batchHeaders = [{
+      batchValue: 200,
+      exportDate: '2021-08-12',
+      ledger: 'AP',
+      numberOfPaymentRequests: 2,
+      sequence: '0001',
+      sourceSystem: 'SFIP'
+    }]
```

Comment:
should these explicitly be referred to as SFI Pilot objects for `schemeType` and `batchHeaders`?